spec/lib/validators/csp_script_src_validator_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+RSpec.describe CspScriptSrcValidator do
+  describe "#valid_value?" do
+    let(:validator) { described_class.new }
+
+    it "returns true for empty string" do
```

Comment:
We could group examples with contexts and simplify things a bit by using the default subject (which does `described_class.new` by default):
```ruby
context "when values are valid" do
  context "when value is an empty string" do
    it { is_expected.to be_a_valid_value "" }
  end

  context "when there's a single value" do
    it { is_expected.to be_a_valid_value "'unsafe-eval'" }
  end

  …
end

context "when values are not valid" do
  context "when there is at least one invalid value" do
    it { is_expected.not_to be_a_valid_value "'unsafe-eval'|'md5-not+a+supported+hash+algo'" }
  end

  …
end
```
Wdyt?
(`be_a_valid_value` is more idiomatic, it reads a bit funny though I guess :sweat_smile:)